articles/synapse-analytics/overview-faq.yml

```diff
@@ -75,14 +75,24 @@ sections:
 
       - question: |
-          Does Azure Synapse workspace Support CI/CD? 
+          What are the functional differences between dedicated SQL pools and serverless pools?
+        answer: |
+          Capabilities and requirements are different between the two types of pools. Differences include database objects, query language capabilities, security, tools, data access, and data format. For detailed comparison of SQL pools and serverless pools, please visit [Pools Comparison](/azure/synapse-analytics/sql/overview-features). For best practices when using either type of pool, please view [Best Practices for Dedicated SQL Pool](/azure/synapse-analytics/sql/best-practices-dedicated-sql-pool) and [Best Practices for Serverless SQL Pool](/azure/synapse-analytics/sql/best-practices-serverless-sql-pool).
+
+      - question: |
+          What are Delta tables, and why should I use them?
+        answer: |
+          Lakehouse is based on open direct-access data formats, such as Apache Parquet. It has first-class support for machine learning and data science. A Delta table is a view of data that is contained in a Delta Lake, which supports most options provided by Apache Spark DataFrame read and write APIs. Lakehouses can help with major challenges with data warehouses, like data staleness, reliability, total cost of ownership, and data lock-in. On Delta tables, optimizations like auto compaction and adaptive query plans are available. For a detailed guide to Delta Lake, please visit [Delta Lake Guide](https://github.com/Azure-Samples/Synapse/blob/main/Notebooks/Scala/Hitchikers%20Guide%20to%20Delta%20Lake%20-%20Scala.ipynb).
+
+       - question: |
+          What is Auto Compaction?
         answer: |
-          Yes! All Pipeline artifacts, notebooks, SQL scripts, and Spark job definitions will reside in Git. All pool definitions will be stored in Git as Azure Resource Manager (ARM) templates. Dedicated SQL pool objects (schemas, tables, views, etc.) will be managed with database projects with CI/CD support.
+          Auto Compaction is one of two complementary features of Auto Optimize for Delta tables. After a write to a table succeeds, Auto Compaction can further compact files for partitions that have the most number of small files. Opting in to Auto Compaction is recommended for streaming use cases where adding minutes of latency is acceptable and when you don't have regular OPTIMIZE calls on your table. For more information on Auto Optimize and Auto Compaction, check out this [Auto Optimize Guide](https://docs.microsoft.com/en-us/azure/databricks/delta/optimizations/auto-optimize).
```

Comment:
@nick-shapiro This is databricks documentation, can you please update Synapse's or remove the link if Synapse does not have one?